lib/site-inspector/headers.rb

```diff
@@ -46,19 +46,14 @@ def xss_protection?
   end
 
   def secure_cookies?
-    return nil if !response || !has_cookies?
-    cookie = header_from("Set-Cookie")
+    return nil if !has_cookies?
+    cookie = headers["set-cookie"]
     cookie = cookie.first if cookie.is_a?(Array)
-    marked_secure = !!(cookie.downcase =~ /secure/)
-    marked_http_only = !!(cookie.downcase =~ /httponly/)
-    marked_secure and marked_http_only
+    !!(cookie.downcase =~ /secure/) && !!(cookie.downcase =~ /httponly/)
```

Comment:
Fixed via  ef67f82 based on https://www.owasp.org/index.php/HttpOnly#What_is_HttpOnly.3F and sanity tested via the CFPB fixture (which sets a secure cookie)